index.bs

```diff
@@ -1767,6 +1786,12 @@ a [=source type=] |sourceType|, and a [=duration=] |expiry|:
 1. If |map|["`event_report_windows`"] does not [=map/exist=], return the result
     of [=obtaining default effective windows=] given |sourceType|, |sourceTime|,
     and |expiry|.
+1. Return the result of [=parsing report windows ignoring legacy=] with |map|,
+    |sourceTime|, and |expiry|.
+
+To <dfn>parse report windows ignoring legacy</dfn> given a [=map=] |map|, a
+[=moment=] |sourceTime|, and a [=duration=] |expiry|:
+
```

Comment:
On line 1798, shouldn't ` |map|["`start_time`"] [=map/exists=]` be ` |values|["`start_time`"] [=map/exists=]`?

Reply:
Resolved in #1077.